cvpysdk/sedstype.py

```diff
@@ -0,0 +1,35 @@
+
+class SEDSType(object):
```

Comment:
This will be used in multiple files, and hence the separate class in a separate file